src/components/PageReady/index.test.js

```diff
@@ -0,0 +1,142 @@
+import React, { Fragment } from 'react';
+import { mount } from 'enzyme';
+import moxios from 'moxios';
+import 'test-util/setup';
+import PageReady, { Context } from './index';
+import defaultRequest from './request';
+
+describe('Page Ready', () => {
+  beforeEach(() => {
+    moxios.install();
+  });
+
+  // eslint-disable-next-line no-undef
```

Comment:
add 'afterEach' to the eslint definitions

Reply:
done